main.nf

```diff
@@ -575,6 +576,7 @@ if(params.aligner == 'star'){
         script:
         prefix = reads[0].toString() - ~/(_R1)?(_trimmed)?(_val_1)?(\.fq)?(\.fastq)?(\.gz)?$/
         def avail_mem = task.memory == null ? '' : "--limitBAMsortRAM ${task.memory.toBytes() - 100000000}"
+        RG = params.seqCenter ? "--outSAMattrRGline ID:$prefix 'CN:$params.seqCenter'" : ''
```

Comment:
Small thing, but I'd prefer a more human readable variable name here. Also I'm not sure if groovy does something special with variable names that are all caps lock - the GitHub syntax highlighting puts it in a different colour to other variable names. I'd prefer for example:

```groovy
def seqcenter = params.seqCenter ? "--outSAMattrRGline ID:$prefix 'CN:$params.seqCenter'" : ''
```